cmd/zed/context.go

```diff
@@ -77,18 +77,33 @@ func contextListCmdFunc(cmd *cobra.Command, args []string) error {
 		secret := token.APIToken
 		if !cobrautil.MustGetBool(cmd, "reveal-tokens") {
 			prefix, _ := token.SplitAPIToken()
-			secret = stringz.Join("_", prefix, "<redacted>")
+			if prefix == "" {
+				secret = "<redacted>"
+			} else {
+				secret = stringz.Join("_", prefix, "<redacted>")
+			}
+		}
+
+		insecure := false
+		if token.Insecure != nil {
+			insecure = *(token.Insecure)
+		}
+
+		insecureStr := ""
+		if insecure {
+			insecureStr = "   ✓    "
```

Comment:
We should make this a constant now that two things are using it

Reply:
They are actually slightly different in length